src/routes/(inner)/utilities/toasts/+page.svelte

```diff
@@ -54,10 +54,17 @@
 		toastStore.trigger({ message: 'Message will auto-hide after 5 seconds.' });
 	}
 
+	function toastPreset(preset: 'primary' | 'accent' | 'tertiary' | 'warning' | 'success' | 'error'): void {
```

Comment:
```suggestion
	function toastPreset(preset: ToastSettings['preset']): void {
```
This'll just nicely tie the type of the preset to the type of the parameter

Reply:
I had no idea this was possible, very useful. Thanks!